miso-web/src/main/webapp/scripts/hot_library.js

```diff
@@ -66,7 +65,7 @@ HotTarget.library = (function() {
           });
           setData(data);
         } else if (flatProperty === 'index' + n + 'Label'){
-          var pt = getPlatformType(flat.platformType);
+          var pt = HotUtils.getPlatformType(flat.platformType);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Rename "pt" as this name is already used in declaration at line 47. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3AS2814)